### PR TITLE
Flutter-3.10

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: lib/src/sdk.dart [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Sdk {
     // The Flutter web SDK path changed. The SDK version test here will go
     // through the master, beta, stable, then old waterfall. Well, the last step
     // is the removal of this test, but you get the idea.
-    if (masterChannel || betaChannel) {
+    if (masterChannel || betaChannel || stableChannel) {
```
Comment (johnpryan) [inline lib/src/sdk.dart:98]: probably dev channel too? Maybe we can take out this if statement now...

**Review by johnpryan — APPROVED** [review]

File: lib/src/sdk.dart (hunk 1 above)
Comment (parlough) [reply lib/src/sdk.dart:98]: I believe `master` is `dev`, and I guess `old` still needs the old behavior?

Comment (domesticmouse) [reply lib/src/sdk.dart:98]: We can take it out of `old` ... in three months.

**Comment by parlough** [comment]
Actions has really been a pain today :/

**Comment by domesticmouse** [comment]
> Actions have really been a pain today :/

I'm guessing everything broke all at once...

**Pull request merged**